controllers/notes.js

```diff
@@ -19,12 +48,42 @@ exports.deletingNote = (req, res, next) => {
 })
 }
 
-exports.createNote = (req, res, next) => {
-    res.status(200).json(
-    req.body //why we don't have success: true here? out 
-    );
+exports.deleteNote = (req, res, next) => {
+  let user = req.user;
+  Note.findByIdAndDelete(req.params.id, ()=>{
+    if (err) {
+      console.log(err);
+    } else {
+      user.notes.pull(req.params.id); //we removed the element from the arr
+      user.save(); //we should save now
+      console.log('User has been saved');
+      //TBD is that my file is To be defined, i havent one yet
+      res.render('views/TBD.hbs')
+    }
+  });
+  req.user.notes;
 }
 
+exports.createNote = (req, res, next) => {
+    let note = new Note(req.body);
```

Comment:
What happens if your req.body.image is null? Is it going to create a note with your default image url?